function handleCall() {
    try {

    }   catch(err) {

    }
}

export default handleCall;